.github/workflows/php.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       matrix:
         operating-system: [ubuntu-latest]
-        php-versions: ['8.0', '8.1', '8.2', '8.3']
+        php-versions: ['8.0', '8.1', '8.2', '8.3', '8.4']
```

Comment:
Did you change your mind about creating a PR to add php 8.4 in the workflow ?

Reply:
All the questions I was going to ask on that new PR, you already answered here 😅
I can still split into a separate PR if you'd like.

Reply:
I am ok to add php 8.4 here but you will have conflict on the  PR #108

Reply:
I'll rebase after that PR